### PR TITLE
Added Disjoint Sets in Python

### PR DESCRIPTION
**Pull request by kaustubh-pandey**
An object-oriented python implementation of Disjoint Sets Datastructure

### CODE REVIEW

**Comment by kaustubh-pandey** [comment]
This is in reference to the issue #481

File: Disjoint_Sets/disjointSets.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+if(d_set.find(4)==d_set.find(6)):
+    print("YES")
+else:
+    print("NO")
+
+# check if 1 and 4 are in the same set
+if(d_set.find(1)==d_set.find(4)):
+    print("YES")
+else:
+    print("NO")
```
Comment (jainaman224) [inline Disjoint_Sets/disjointSets.py]: Add new line.

**Comment by jainaman224** [comment]
Please fix file name.

**Comment by kaustubh-pandey** [comment]
I have fixed the file name and added newline

**Pull request merged**